kubernetes/templates/5-prometheus.yaml

```diff
@@ -52,12 +52,20 @@ metadata:
 spec:
   groups:
   - name: DemoAlerts
+    # Note: all the rules below need to include the endpoint, namespace, and job
+    # labels to ensure the EDA Pod receives the firing events. Maybe this can be
+    # configured/filtered using some filtering logic on the alertmanager YAMLs?
     rules:
     - alert: InstanceDown 
-      expr: up{job="services"} < 1 
+      # If the replica count of the deployment drops below the given threshold,
+      # scale the replica count up.
+      expr: count by(job, endpoint, service, container, namespace) (up{job="quarkus-monitor"}) < 2
```

Comment:
The labels included using `by` seem to be necessary to ensure alerts are sent to EDA.